docs/configuration.md

```diff
@@ -264,6 +264,7 @@ domain must be separated by a comma.
 | customTTL           | duration (no unit is minutes)               | no        | 1h            |
 | rewrite             | string: string (domain: domain)             | no        |               |
 | mapping             | string: string (hostname: address or CNAME) | no        |               |
+| zoneFileMapping     | multiline string containing a DNS Zone File | no        |               |
```

Comment:
Since it supports already more than the simple mapping and is no file by itself I would opt for ` zone`  as config key. 🤔